### PR TITLE
Add icons for three dots menu and cogwheel

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add icons for three dots menu and cogwheel

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Enhanced UI by replacing textual representations of the three dots menu with vertical dots icons across various documentation pages for improved visual clarity and consistency.
  - Updated icons for cogwheel and other elements to improve user interaction and aesthetic appeal.

- **Documentation**
  - Updated multiple documentation files to reflect these changes, ensuring that visual guides are clear and consistent for end-users navigating the Home Assistant interface.

### CODE REVIEW

File: source/_includes/common-tasks/beta_version.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ If you would like to test next release before anyone else, you can install the b
     4. Go to the {% my configuration title="**Configuration**" %} panel.
     5. Install the update that is presented to you.
        - **Troubleshooting**: If you do not see that notification:
-         - In the top right corner, select the three dots menu and select **Check for updates**.
+         - In the top right corner, select the three dots <iconify-icon inline icon="mdi:dots-vertical"></iconify-icon> menu and select **Check for updates**.
```
[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/beta_version.md]: in future, please do not comment on HTML for icons

File: source/dashboards/dashboards.markdown [hunk 2]
```diff
@@ -181,7 +181,7 @@ mode:
   type: string
 resources:
   required: false
-  description: "List of resources that should be loaded. Only use this when mode is `yaml`. If you change anything here, click the three dots menu (top-right) and click on `Reload resources` to pick up changes without restarting Home Assistant. You can also call `lovelace.reload_resources` service directly."
+  description: "List of resources that should be loaded. Only use this when mode is `yaml`. If you change anything here, click the three dots <iconify-icon inline icon="mdi:dots-vertical"></iconify-icon> menu (top-right) and click on `Reload resources` to pick up changes without restarting Home Assistant. You can also call `lovelace.reload_resources` service directly."
```
Comment (c0ffeeca7) [inline source/dashboards/dashboards.markdown]:
```suggestion
  description: "List of resources that should be loaded. Only use this when mode is `yaml`. If you change anything here, click the three dots menu (top-right) and click on `Reload resources` to pick up changes without restarting Home Assistant. You can also call `lovelace.reload_resources` service directly."
```

File: source/voice_control/android.markdown [hunk 3]
```diff
@@ -27,7 +27,7 @@ To define Home Assistant Assist as default assistant app, follow these steps:
 2. Under **Digital assistant app**, select **Default digital assistant app**.
    - On some Android versions, this might be under **Digital & voice input** > **Default digital assistant app** > **Voice Assistant**.
 3. Select **Home Assistant**.
-   - On some Android versions, you might be able to change the assistants language settings here by selecting the cogwheel.
+   - On some Android versions, you might be able to change the assistants language settings here by selecting the cogwheel <iconify-icon inline icon="mdi:cog-outline"></iconify-icon>.
```
Comment (c0ffeeca7) [inline source/voice_control/android.markdown]:
```suggestion
   - On some Android versions, you might be able to change the assistant's language settings here by selecting the cogwheel <iconify-icon inline icon="mdi:cog-outline"></iconify-icon>.
```

File: source/_includes/common-tasks/beta_version.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (c0ffeeca7) [reply source/_includes/common-tasks/beta_version.md]: can you summarize what you have learned?

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**